src/System.Data.SqlClient/src/System/Data/SqlTypes/SQLResource.cs

```diff
@@ -14,66 +14,64 @@ namespace System.Data.SqlTypes
 {
     using Res = System.SR;
 
-    internal sealed class SQLResource
+    internal static class SQLResource
     {
-        private SQLResource() { /* prevent utility class from being instantiated*/ }
+        internal static string NullString => Res.GetString(Res.SqlMisc_NullString);
```

Comment:
While you're at it, might as well remove the calls to `Res.GetString` in each of these properties as all it does is return the passed-in string.

As an aside, not necessarily for this PR, but these properties on `SQLResource` could probably be removed altogether, updating all the call sites to access `Res.Foo` directly instead of through `SQLResource`.